google/cloud/storage/_helpers.py

```diff
@@ -326,12 +327,21 @@ def patch(
         :type retry: google.api_core.retry.Retry or google.cloud.storage.retry.ConditionalRetryPolicy
         :param retry:
             (Optional) How to retry the RPC. See: :ref:`configuring_retries`
+
+        :type override_unlocked_retention: bool
+        :param override_unlocked_retention:
+            (Optional) Applicable for objects that have an unlocked retention configuration.
```

Comment:
I can't say I fully understand this explanation yet. Is it in line with other language docstrings?

Reply:
I agree it's quite confusing. As part of the feature, a new boolean parameter `overrideUnlockedRetention` is added to bucket patch and update operations, as shown in the [official documentation](https://cloud.google.com/storage/docs/json_api/v1/objects/patch#parameters). 

Would it be more clear to update the docstrings to something along the lines of:
```
overrideUnlockedRetention must be set to true if the operation includes a
retention property that changes the mode from Unlocked to Locked, 
reduces the retainUntilTime, or removes the retention configuration from the object.
```